packages/daimo-api/src/contract/coinIndexer.ts

```diff
@@ -63,9 +63,7 @@ export class CoinIndexer {
           v as "value"
         from transfers
         where block_num >= $1
-        and block_num <= $2
-        and chain_id = $3
-        and log_addr = $4;
```

Comment:
checked `select
          block_num,
          block_hash,
          tx_hash,
          tx_idx,
          log_idx,
          log_addr,
          f as "from",
          t as "to",
          v as "value"
        from transfers
        where log_addr <> '\x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913';` is empty

Reply:
and same for chain ID